addons/l10n_it/models/account_chart_template.py

```diff
@@ -10,6 +10,11 @@ def _load(self, company):
         """ Set tax calculation rounding method required in Italian localization
         Also to avoid rounding errors when sent with FatturaPA"""
         res = super()._load(company)
+        self = self.with_company(company)
         if company.account_fiscal_country_id.code == 'IT':
             company.write({'tax_calculation_rounding_method': 'round_globally'})
+            vat_split_payment_account = self.env['account.account'].search([('company_id', '=', company.id), ('code', 'like', '2607%')])
+            split_payment_tax_group = self.env.ref('l10n_it.tax_group_split_payment')
```

Comment:
```suggestion
            split_payment_tax_group = self.env.ref('l10n_it.tax_group_split_payment').with_company...
```

Reply:
You sure? `self` is already `self.with_company()` since line 4, so when doing the `ref` it should still be there... ?

Reply:
It will work.  I suggested for clarity.